src/test/java/net/kaczmarzyk/spring/data/jpa/swagger/springdoc/SpecificationArgResolverSpringdocOperationCustomizerTest.java

```diff
@@ -306,7 +426,7 @@ public void disjunctionTestMethod(
 				value = @And({
 					@Spec(path = "", params = "disjunctionSecondParam", spec = Like.class),
 					@Spec(path = "", params = "disjunctionThirdParam", spec = Like.class)
-				})) Specification<Customer> spec) {
+				})) Specification<Object> spec) {
```

Comment:
why?

Reply:
Tests from that class were not using customer class at all, so I decided to replace it with something more generic.